### PR TITLE
Upload files

### PR DESCRIPTION
**Pull request by MarjorieThomassin**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,9 @@
     "dotenv": "^10.0.0",
     "express": "^4.17.1",
     "jsonwebtoken": "^8.5.1",
-    "mysql2": "^2.2.5"
+    "multer": "^1.4.2",
+    "mysql2": "^2.2.5",
+    "nodemon": "^2.0.7"
```
Comment (rocambille) [inline package.json]: nodemon should be in dev dependencies: uninstall it, and reinstall it with --save-dev

File: routes/upload.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+const multer = require('multer');
+
+const upload = multer({ dest: 'tmp/' });
+
+const fs = require('fs');
+
+// define the index route
+uploadRoute.post('/', upload.fields([{ name: 'bottleFront' }, { name: 'bottleBack' }]), (req, res) => {
+  const front = req.files.bottleFront[0];
+  console.log(front);
```
Comment (rocambille) [inline routes/upload.js]: you can remove the console logs :)

File: routes/upload.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    fs.rmSync(front.path);
+    stringFront = 'L\'image recto est trop volumineuse';
+  }
+  if (back.size <= 100000) {
+    fs.renameSync(back.path, `uploads/${back.originalname}`);
+  } else {
+    fs.rmSync(back.path);
+    stringBack = 'L\'image verso est trop volumineuse.';
+  }
+  res.send(`${stringFront}. ${stringBack}`);
```
Comment (rocambille) [inline routes/upload.js]: good start, but your front will expect json + you're not sending a proper status for an error

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by rocambille** [comment]
you should remove the upload directory from your PR, and add it to .gitignore ;)

File: routes/upload.js (hunk 2 above)
Comment (MarjorieThomassin) [reply routes/upload.js]: fixed

File: package.json (hunk 1 above)
Comment (MarjorieThomassin) [reply package.json]: fixed

File: routes/upload.js (hunk 3 above)
Comment (MarjorieThomassin) [reply routes/upload.js]: fixed

**Comment by rocambille** [comment]
> you should remove the upload directory from your PR, and add it to .gitignore ;)

remember this @MarjorieThomassin

**Comment by MarjorieThomassin** [comment]
@rocambille it's deleted :)

**Pull request merged**